src/block/block_ext.c

```diff
@@ -194,7 +205,7 @@ __block_ext_insert(WT_SESSION_IMPL *session, WT_EXTLIST *el, WT_EXT *ext)
         /*
          * Insert the new WT_EXT structure into the size element's offset skiplist.
          */
-        __block_off_srch(szp->off, ext->off, astack, true);
```

Comment:
Could it be possible to have a new element at the end of the list through `__block_ext_insert`? I have noticed we pass `NULL` here and below for the `penultimate_extp` variable.